requirements_all.txt

```diff
@@ -300,10 +300,12 @@ blockchain==1.4.4
 # homeassistant.components.bom
 bomradarloop==0.1.3
 
-# homeassistant.components.amazon_polly
 # homeassistant.components.route53
 boto3==1.9.16
 
+# homeassistant.components.amazon_polly
+boto3==1.9.233
```

Comment:
Could you please update route53 to use `boto3==1.9.233` as well?

Reply:
done. Looks like things are moving along. I'll work on the docs.